### DISCUSSION TITLE
Documentation target picker perhaps not as obvious as we'd like

### DISCUSSION THREAD

**Discussion by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#2177] [General]
I feel like we've discussed this before somewhere but couldn't find any issues or discussions.

In a recent PR, [someone commented](https://github.com/apple/swift-distributed-tracing/pull/72#issuecomment-1337035952):

> it takes you to the Instrumentation documentation instead of Tracing without any UI to switch to the Tracing documentation

I actually initially overlooked this comment and then while helping with the PR ended up struggling to figure out if my fix worked, because [I couldn't find the target picker](https://github.com/apple/swift-distributed-tracing/pull/72#issuecomment-1337907951) 🙈

So I think there's something there that may need addressing.

It may be as simple as adding a drop-down chevron to indicate the breadcrumb is a menu here:

<img width="347" alt="CleanShot 2022-12-06 at 10 56 25@2x" src="https://user-images.githubusercontent.com/65520/205879002-341a0038-40f2-40b5-a62b-d6a4b51d824d.png">

(and to the version segment as well while we're at it).

**Comment by daveverwer** [comment]
#2022 was the original discussion. Let's promote it to an issue so it doesn't get lost.

I was really not happy with the double disclosure triangles, but you're right it's a problem. I'll get it fixed.
